pulsar-common/src/main/java/org/apache/pulsar/common/api/raw/RawMessageId.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.pulsar.common.api.raw;
 
+@SuppressWarnings("checkstyle:JavadocType")
```

Comment:
Any reason why we are suppressing Javadoc warning? I think it is better to fix the java doc since we are touching the files.

Reply:
No particular reason, but writing javadoc comments takes a considerable amount of time. The problem was that this module had a lot of violations (appx. 600) and it was hard to fix all of them at once. I fixed most of the suppressed warnings related to missing javadoc comments. In some cases, I could not find fitting descriptions and I left the suppressed warnings.